specs/latest/2.0/index.html

```diff
@@ -1274,6 +1274,9 @@ <h2 class="no-toc">Table of contents</h2>
         framebuffer. If the value of SAMPLE_BUFFERS for the read framebuffer is one and the value of
         SAMPLE_BUFFERS for the draw framebuffer is zero, the samples corresponding to each pixel location in
         the source are converted to a single sample before being written to the destination.
+
+        For any source pixel lying outside the read framebuffer, the corresponding destination pixel remains
+        untouched.
```

Comment:
It's a good question. Looking back at the old Khronos bug https://cvs.khronos.org/bugzilla/show_bug.cgi?id=3576 (sorry, non-public) linked from http://crbug.com/644740 , defining the math for the coordinate system transformations seemed difficult, and the OpenGL ES working group themselves opted for simpler spec language.

@Kangz do you have any suggestion about how to phrase this?

There does seem to be some ambiguity about what happens with pixels on the border of the source rectangle. For a LINEAR blit, CLAMP_TO_EDGE filtering should presumably be used, so that even if the source rectangle is fully inside the source framebuffer, pixels on the other side aren't referenced at all.